packages/devtools_app/lib/src/inspector/layout_explorer/flex/flex.dart

```diff
@@ -73,64 +70,112 @@ const minHeightToAllowTruncating = 375.0;
 const minWidthToAllowTruncating = 375.0;
 
 // Story of Layout colors
```

Comment:
these colors were actually changed as part of the layout bug. This file is the only file with actual code changes. Trying to keep them together to avoid merge conflicts. The layout explorer was previously broken by ThemedColor combining with the color scheme not being fixed for the duration of running the app so it was hard to test this without fixing the theme issue.